tests/toolchain/gcc5_C_compilation.pm

```diff
@@ -0,0 +1,42 @@
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    type_string "wget https://github.com/linux-test-project/ltp/releases/download/20150420/ltp-full-20150420.tar.bz2 && echo 'OK'\n";
+    type_string "tar jxf ltp-full-20150420.tar.bz2\n";
+    sleep 2;
+    type_string "export CC=/usr/bin/gcc-5\n";
+    type_string "export CXX=/usr/bin/g++-5\n";
+    type_string "cd ltp-full-20150420\n";
+    sleep 2;
+    type_string "setterm -blank 0\n";   # disable screensaver
+    type_string "./configure --with-open-posix-testsuite && echo 'configure PASSED-$?'|tee /dev/$serialdev\n";
```

Comment:
Updated with `assert_script_run` and moved packages to data instead of always downloading them

http://10.100.98.90/tests/1612